modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -6,14 +6,16 @@
 [id="sbo-binding-workloads-that-are-not-compliant-with-PodSpec_{context}"]
 = Binding secondary workloads that are not compliant with PodSpec
 
-[role="_abstract"]
 A typical scenario in service binding involves configuring the backing service, the workload (Deployment), and {servicebinding-title}. Consider a scenario that involves a secondary workload (which can also be an application Operator) that is not compliant with PodSpec and is between the primary workload (Deployment) and {servicebinding-title}.
 
 For such secondary workload resources, the location of the container path is arbitrary. For service binding, if the secondary workload in a CR is not compliant with the PodSpec, you must specify the location of the container path. Doing so projects the binding data into the container path specified in the secondary workload of the `ServiceBinding` custom resource (CR), for example, when you do not want the binding data inside a pod.
 
-{servicebinding-title} provides an option to configure the value of where the container path or secret path is and bind these paths at a custom location. This option is available only for the `binding.operators.coreos.com` API group when the binding data is projected as environment variables.
+{servicebinding-title} provides an option to configure the value of where the container path or secret path is and bind these paths at a custom location.
 
+[id="configuring-custom-location-of-container-path_{context}"]
 == Configuring the custom location of the container path
+This option is available only for the `binding.operators.coreos.com` API group when the binding data is projected as environment variables.
```

Comment:
- The word `only` is playing games with my understanding of this sentence. Can we remove `only` and keep the same meaning?
- Can we restate the name of the option, custom location, instead of saying "This option"?
- Can we rephrase "is projected" using the active voice?

How about:
```suggestion
This custom location is available for the `binding.operators.coreos.com` API group when {servicebinding-title} projects the binding data as environment variables.
```

Reply:
Can you rephrase this abstract in terms of the user's goals and actions? You can typically produce content that is much more user-centered by applying the following template to the first sentence in each section:
`To <goal>, you can <action> by using <feature>.`

Reply:
Fixed